src/services/user/hooks/publicTeachers.js

```diff
@@ -4,9 +4,16 @@ const { disallow } = require('feathers-hooks-common');
 
 const { Forbidden } = require('../../../errors');
 const globalHooks = require('../../../hooks');
-const { lookupSchool } = require('../../../hooks');
 const { equal: equalIds } = require('../../../helper/compare').ObjectId;
 
+const restrictToTeachersWithTeams = async (hook) => {
+	const permited = await globalHooks.hasRole(hook, hook.params.account.userId, 'teacher');
```

Comment:
maybe we should also include administrator? I am not sure, but what if ins some schools the administrator is also a teacher, and is using the same account?

Reply:
It is fine we want teacher and teacher+admin, but not admin without teachers as role.